### PR TITLE
Restore command

### PR DESCRIPTION
**Pull request by al-kau**

### CODE REVIEW

File: src/Eppie.CLI/Eppie.CLI/Resources/Program.resx [hunk 1]
```diff
@@ -230,6 +238,14 @@
     <value>Enter your new vault password: </value>
     <comment>This message prompts the user to enter a new vault password.</comment>
   </data>
+  <data name="Message.AskRestorePath" xml:space="preserve">
+    <value>Enter the restore URI: </value>
+    <comment>This message prompts the user to enter the path to the backup.</comment>
+  </data>
+  <data name="Menu.RestoreDescription" xml:space="preserve">
+    <value>Restore your master account</value>
```
Comment (ValeraFinebits) [inline src/Eppie.CLI/Eppie.CLI/Resources/Program.resx:246]: Could you please change to Eppie account?

File: src/Eppie.CLI/Eppie.CLI/Resources/Program.resx [hunk 2]
```diff
@@ -222,6 +222,14 @@
     <value>Accounts:</value>
     <comment>This is the header for the list of accounts.</comment>
   </data>
+  <data name="Message.AppRestored" xml:space="preserve">
+    <value>The application has been restored successfully.</value>
```
Comment (ValeraFinebits) [inline src/Eppie.CLI/Eppie.CLI/Resources/Program.resx:226]: Could you please change to Eppie account restored?

File: src/Eppie.CLI/Eppie.CLI/Services/Application.cs [hunk 3]
```diff
@@ -218,6 +232,12 @@ internal void WriteApplicationOpenedMessage()
             Console.WriteLine(_resourceLoader.Strings.AppOpened);
         }
 
+        internal void WriteApplicationRestoredMessage()
+        {
+            _logger.LogDebug("The application was restored.");
```
Comment (ValeraFinebits) [inline src/Eppie.CLI/Eppie.CLI/Services/Application.cs:237]: Could you please change to Eppie account restored?

**Review by ValeraFinebits — CHANGES_REQUESTED** [review]

File: src/Eppie.CLI/Eppie.CLI/Resources/Program.resx (hunk 2 above)
Comment (al-kau) [reply src/Eppie.CLI/Eppie.CLI/Resources/Program.resx]: done at [commit](https://github.com/Eppie-io/Eppie-CLI/pull/82/commits/392e5e3a11abf4947162a6a833ac17e0319efa6a)

File: src/Eppie.CLI/Eppie.CLI/Resources/Program.resx (hunk 1 above)
Comment (al-kau) [reply src/Eppie.CLI/Eppie.CLI/Resources/Program.resx]: done at [commit](https://github.com/Eppie-io/Eppie-CLI/pull/82/commits/392e5e3a11abf4947162a6a833ac17e0319efa6a)

**Review by ValeraFinebits — APPROVED** [review]

**Pull request merged**